### PR TITLE
mark internal functions

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: R/install_easystats.R [hunk 1]
```diff
@@ -40,7 +40,7 @@ install_latest <- function(source = c("development", "cran"),
   source <- match.arg(source, c("development", "cran"))
   pkg <- .packages_on_cran()
 
-  if (all(packages == "all")) {
+  if (packages == "all") {
```
Comment (strengejacke) [inline R/install_easystats.R:43]: packages can be vector length > 1, so this will warn/error.

Comment (IndrajeetPatil) [reply R/install_easystats.R:43]: I don't understand this. 

If `packages` can be a character vector, won't this _always_ to evaluate to `FALSE`? 

```r
(packages == "all")
```

Comment (IndrajeetPatil) [reply R/install_easystats.R:43]: Oh, I see, I get it.

Comment (IndrajeetPatil) [reply R/install_easystats.R]: I have changed the code to make it easier to reason about.

**Review by strengejacke — APPROVED** [review]

**Pull request merged**